examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3182,19 +3194,7 @@ var GLTFLoader = ( function () {
 					// Overrides .createInterpolant in a factory method which creates custom interpolation.
 					if ( sampler.interpolation === 'CUBICSPLINE' ) {
 
-						track.createInterpolant = function InterpolantFactoryMethodGLTFCubicSpline( result ) {
-
-							// A CUBICSPLINE keyframe in glTF has three output values for each input value,
-							// representing inTangent, splineVertex, and outTangent. As a result, track.getValueSize()
-							// must be divided by three to get the interpolant's sampleSize argument.
-
-							return new GLTFCubicSplineInterpolant( this.times, this.values, this.getValueSize() / 3, result );
-
-						};
-
-						// Workaround, provide an alternate way to know if the interpolant type is cubis spline to track.
-						// track.getInterpolation() doesn't return valid value for custom interpolant.
-						track.createInterpolant.isInterpolantFactoryMethodGLTFCubicSpline = true;
+						track.createInterpolant = GLTFCubicSplineInterpolant.FactoryMethod;
```

Comment:
It's fine if you want to, but you don't need to include changes to `examples/jsm` at this time. The copy in `examples/js` is enough and the change will be carried through later.

Reply:
Ok